src/core/mac/mac.cpp

```diff
@@ -269,7 +269,11 @@ ThreadError Mac::SetNetworkName(const char *aNetworkName)
     VerifyOrExit(strlen(aNetworkName) <= OT_NETWORK_NAME_MAX_SIZE, error = kThreadError_InvalidArgs);
 
     memset(&mNetworkName, 0, sizeof(mNetworkName));
+#ifdef _WIN32
+    strcpy_s(mNetworkName.m8, sizeof(mNetworkName), aNetworkName);
+#else
```

Comment:
`strlcpy` doesn't seem to work. It doesn't exist on Windows and when I compile on Linux:
```
mac/mac.cpp: In member function ‘ThreadError Thread::Mac::Mac::SetNetworkName(const char*)’:
mac/mac.cpp:272:64: error: ‘strlcpy’ was not declared in this scope
     strlcpy(mNetworkName.m8, aNetworkName, sizeof(mNetworkName));
```